apps/prairielearn/src/server.js

```diff
@@ -1847,13 +1871,21 @@ if (config.startServer) {
       async () => {
         logger.verbose('PrairieLearn server start');
 
-        let configFilename = 'config.json';
+        // For backwards compatibility, we'll try loading a `config.json` file
+        // in both the repository root and in the `apps/prairielearn` directory.
+        let configFilenames = [
+          path.resolve(__dirname, '..', 'config.json'),
+          path.resolve(__dirname, '..', '..', '..', 'config.json'),
+        ];
```

Comment:
👀 do we think this is actually a good idea? I think it's good to avoid breaking people's dev workflows. If we like this, I'll add `apps/prairielearn/config.json` to `.gitignore`.

Reply:
I should probably do this for graders + workspaces too - especially workspaces, as they historically read from `config.json` in the root of the repo.

Reply:
Yeah, I think preserving people's dev workflows is a very polite thing to do.

Reply:
I'm doing this in #7667 - `.gitignore` change to come!

Reply:
`.gitignore` adjusted in 994455e,